src/components/ContactUs/Map/Map.js

```diff
@@ -0,0 +1,43 @@
+import ReactMapboxGl, {
+  Marker,
+  ScaleControl,
+  ZoomControl,
+  RotationControl,
+} from 'react-mapbox-gl';
+import 'mapbox-gl/dist/mapbox-gl.css';
+
+function Map() {
+  const MapDetail = ReactMapboxGl({
+    accessToken:
+      'pk.eyJ1IjoidGhlYWtpbmthdmFrIiwiYSI6ImNsMWwzY21qYjAxdDczYnBxMDllY2k3YXEifQ.UPpP7OQtEb-wfVWnxqxV0A',
+  });
+
+  return (
+    <div
+      className="flex text-center lg:text-center items-center justify-center"
+      data-testid="valid-map"
+    >
+      <MapDetail
+        // eslint-disable-next-line react/style-prop-object
```

Comment:
Does't this work without this exception ? We don't add exceptions.

Reply:
When I try to compile the next line, I get the following warning: **style prop value must be an object.** When I changed it to an object, the map is not rendering.